app/Phragile/AssigneeCollection.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+namespace Phragile;
+
+class AssigneeCollection {
+
+	private $assignees;
+
+	public function __construct(PhabricatorAPI $phabricator, array $tasks)
+	{
+		$assigneePHIDs = $this->extractUniqueAssignees($tasks);
+		$this->assignees = $this->processAssignees(
+			$phabricator->getUserData(['phids' => $assigneePHIDs])
+		);
```

Comment:
This prevents you from creating the collection if you already have the assignees.